curriculum/challenges/english/08-coding-interview-prep/rosetta-code/fizzbuzz.english.md

```diff
@@ -0,0 +1,92 @@
+---
+title: FizzBuzz
+id: 5e9ddb06ec35240f39657419
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+
+Write a program that outputs the integers from 1 to 100 (inclusive). But: <br>
+<ul>
+    <li> for multiples of 3, output <b>Fizz</b> instead of the number </li>
+    <li> for multiples of 5, output <b>Buzz</b> instead of the number </li>
+    <li> For multiples of 3 and 5, output <b>FizzBuzz</b> instead of the number </li>
+</ul>
+
+</section>
+
+## Instructions
+<section id='instructions'>
+
+Your program should return an array containing the results based on the rules above.
+
+</section>
+
+## Tests
+<section id='tests'>
+
+```yml
+tests:
+  - text: <code>fizzBuzz</code> should be a function.
+    testString: assert(typeof fizzBuzz=='function');
+  - text: <code>fizzBuzz()</code> should return an Array.
+    testString: assert(Array.isArray(fizzBuzz())==true);
+  - text: Number only divisible by 3 should return <code>"Fizz"</code>.
+    testString: assert.equal(fizzBuzz()[50], "Fizz");
+  - text: Numbers only divisible by 5 should return <code>"Buzz"</code>.
+    testString: assert.equal(fizzBuzz()[49], "Buzz");
```

Comment:
Similarly, we can check the last value with this test to be sure that the values in the array go up to 100:

`testString: assert.equal(fizzBuzz()[99], "Buzz");`